form-flow/src/main/kotlin/com/ritense/formflow/domain/instance/FormFlowStepInstance.kt

```diff
@@ -52,7 +52,11 @@ data class FormFlowStepInstance(
     val definition: FormFlowStep
         get() = instance.formFlowDefinition.getStepByKey(stepKey)
 
-    fun back() {
+    fun back(incompleteSubmissionData: String? = null) {
```

Comment:
@marijnritense When the user in the browser click on back, the user will get three options:
- Click 'Yes, save data'. The FE should call `/back` _and_ send the incompleteSubmissionData. 
- Click 'No, don't save data'. The FE should call `/back` but _not_ send any data. The mothod above, will receive `null`
- Click 'Cancel'. The FE closes the popup but shouldn't call `/back`

Reply:
![image](https://user-images.githubusercontent.com/94897451/207829548-320dcf12-ab8e-4701-b999-ac082d99a25c.png)

If you click outside of the form flow, you should get those three options as well, but then it shouldn't call /back as we don't want to go back a step, we just want to be able to continue where we left off next time we open the task.